src/common/services/session/session.service.js

```diff
@@ -203,12 +203,13 @@ const session = /* @ngInject */ function ($cookies, $rootScope, $http, $timeout,
         $window.sessionStorage.setItem(locationOnLogin, $window.location.href)
       }
 
-      return authClient.signOut({
-        postLogoutRedirectUri: redirectHome ? null : `${envService.read('oktaReferrer')}/sign-out.html`
-      }).catch(() => {
-        // If Okta sign out fails, redirect to Okta sign out page
-        $window.location.href = `${envService.read('oktaUrl')}/login/signout?fromURI=${envService.read('oktaReferrer')}`
-      })
+      // We no longer use the below code because it doesn't redirect the user back to the Give site
+      // Instead, it redirects to the Okta sign in page.
+      // return authClient.signOut({
+      //   postLogoutRedirectUri: redirectHome ? null : `${envService.read('oktaReferrer')}/sign-out.html`
+      // })
```

Comment:
I'd prefer not to leave commented out code in the codebase. Is there a reason this can't be removed?

Reply:
It's mainly so we don't try it again, but I can make it smaller

Reply:
That makes sense. Maybe we can delete the code, but leave a comment that says something like "Redirect instead of using authClient.signOut() because signOut leaves the user on the Okta dashboard"